.github/workflows/build-macos.yml

```
@@ -39,7 +39,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v6
         with:
-          go-version: '1.25.1'
+          go-version: '1.25.3'
```

I think this will always use the latest stable release (1.25.x), which I think is a better idea than picking one specific version:

```suggestion
          go-version: '1.25'
```

---

Actually what we want is now this: https://github.com/actions/setup-go?tab=readme-ov-file#using-stableoldstable-aliases

I changed this PR to use `'stable'` and force pushed.

---

Are you sure about this? We'd get auto switched to every next major Go version when it's released and hits `stable` :thinking:

---

I think we do want that, yes. We are already maintaining compatibility and a test failure here would be a valid signal that "something must be done".

---

Oh no please don't use `stable`. It will just suddenly break CI. Yes we want to update TinyGo whether a new stable Go release happens, but sometimes things get delayed and I would really want to have usable CI during that time.

---

In that case, according to the docs, we need to go back to the value `'1.25.3'`. See https://github.com/actions/setup-go?tab=readme-ov-file#supported-version-syntax

---

I checked and `'1.25'` does appear to do what you said @aykevl 

Changed and force pushed.

---

Looked again, and behold:

```
Found in cache @ /Users/runner/hostedtoolcache/go/1.25.1/arm64
```

So `1.25` just uses whichever is cached. I will have to go back to `1.25.3`

---

https://github.com/actions/setup-go?tab=readme-ov-file#check-latest-version
This is probably the best way to ensure the latest release of a specific stable version is used.